presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlCaseInsensitiveMapping.java

```diff
@@ -34,20 +35,17 @@
 public class TestPostgreSqlCaseInsensitiveMapping
         extends AbstractTestQueryFramework
 {
-    private final TestingPostgreSqlServer postgreSqlServer;
+    private TestingPostgreSqlServer postgreSqlServer;
 
-    public TestPostgreSqlCaseInsensitiveMapping()
-    {
-        this(new TestingPostgreSqlServer());
```

Comment:
I guess testng creates all test objects at start and then run the tests from it. So we end up having several postgres docker containers running at once. Am I right?

Reply:
Correct. Which isn't a huge deal for postgres (not resource hungry). Might be a problem elsewhere.

My main motivation is err reporting when test constructor fails (it's far from complete)